### PR TITLE
Added a new element to the Footer and imported the Footer component i…

### PR DESCRIPTION
**Pull request by melo4yxa77796**
OurValues.js: Displays company values .

Screenshot 2024-10-08 at 19 45 09

OurCrew.js: Displays team members with their photos and descriptions.
Screenshot 2024-10-08 at 19 45 28

 OurPartners.js: 
Screenshot 2024-10-08 at 19 46 19

Displays Footer
Screenshot 2024-10-08 at 19 46 29

### CODE REVIEW

File: app/about_us/OurCrew.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import styles from "./crew.module.css";
+
+const OurCrew = () => {
+  const crewFields = [
```
Comment (vdaguenet) [inline app/about_us/OurCrew.js:4]: Excellent idea 👍🏻

File: app/about_us/OurCrew.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+export default OurCrew;
+
+/*const OurCrew = () => [
```
Comment (vdaguenet) [inline app/about_us/OurCrew.js]: Please don't leave dead code in your homework.

**Review by vdaguenet — APPROVED** [review]

File: app/about_us/OurCrew.js (hunk 1 above)
Comment (melo4yxa77796) [reply app/about_us/OurCrew.js:4]: thank you

File: app/about_us/OurCrew.js (hunk 2 above)
Comment (melo4yxa77796) [reply app/about_us/OurCrew.js]: 👍

**Pull request merged**